### DISCUSSION TITLE
MYSQL family: Do Directus go better on MySQL or MariaDB?

### DISCUSSION THREAD

**Discussion by biapar** [directus/directus#9850] [Archive]
MYSQL family: Do Directus go better on MySQL or MariaDB?

**Comment by benhaynes** [comment]
From the Directus standpoint, all vendors are equal. We don't include a lot of vendor conditionals, and focus on feature parity. That said, some vendors intrinsically offer more/less features that may be relevant to your project.

**Reply by biapar** [reply]
Ok. I know well SQL Server but I've to switch to MYSQL because there are some issue ( see issues ) and I've to present a beta version of a project.
